src/core/js/views/drawerView.js

```diff
@@ -211,7 +211,12 @@ class DrawerView extends Backbone.View {
     const direction = {};
     // only trigger popup:closed if drawer is visible
     if (this._isVisible) {
-      Adapt.a11y.popupClosed($toElement);
+      const focusStack = $(Adapt.a11y._popup._focusStack[0]);
+      const $elementToFocus = $toElement || focusStack;
+      if (!$elementToFocus) {
```

Comment:
This condition will never resolve to a truthy.

Reply:
https://github.com/adaptlearning/adapt_framework/blob/master/src/core/js/a11y/popup.js#L139 This is what already happens in `Adapt.a11y.popupClosed` anyway